communication/moderation.md

```diff
@@ -1,11 +1,75 @@
 # Moderation on Kubernetes Communications Channels
 
 This page describes the rules and best practices for people chosen to moderate
-Kubernetes communications channels. This includes: Slack, the mailing lists
-and _any communication tool_ used in an official manner by the project.
+Kubernetes communications channels. This includes: Slack, the mailing lists and
+_any communication tool_ used in an official manner by the project.
```

Comment:
should we say upstream or list them all here? ie - are we responsible for twitter? we need to make the designation here since CNCF owns and/or operates a lot of the properties. we can get the list from the charter.

Reply:
I've revised adding a link to the section of our charter which lists them all, also added a bullet saying CNCF owns some of our properties.